testsuite/run_sets/testsuite.yml

```diff
@@ -67,6 +67,8 @@
 - features/min_docker_build_image.feature
 - features/trad_metadata_check.feature
 - features/srv_clone_channel_npn.feature
+- features/srv_delete_channel.feature
+- features/srv_spacewalk_remove_channel_tool_delete_channel.feature
```

Comment:
Isn't that a name a bit too long?

Reply:
I would say yes, you can reduce it to your last suggestion, as the main purpose of the tool is to remove a channel.

Reply:
done